packages/expo-file-system/src/FileSystem.types.ts

```diff
@@ -1,16 +1,55 @@
+export enum FileSystemSessionType {
+  /*
+   * The session will work even if the user backgrounds an application.
+   *
+   * If a task complete when the application is inactive, the promise might resolve immediately.
+   * However, js code will be stopped after a couple of seconds and it will be resume when the user comes back to the application.
```

Comment:
```suggestion
  /*
   * Using this mode means that the downloading/uploading session on the native side will work even if the application is moved to the `background`.
   *
   * If the task completes when the application is in the `background`, the promise might be resolved immediately.
   * However, javascript execution will be stopped after a couple of seconds and it will be resumed when the application is moved to the `foreground` again.
```